paper/paper.md

```diff
@@ -0,0 +1,337 @@
+---
+title: 'Developing and deploying an integrated workshop curriculum teaching computational skills for reproducible research'
+tags:
+  - reproducible research
+  - data science
+  - R
+  - git
+  - Unix shell
+authors:
+  - name: Zena Lapp^[co-first author]
+    orcid: 0000-0003-4674-2176
+    affiliation: 1
+  - name: Kelly L. Sovacool^[co-first author]
+    orcid: 0000-0003-3283-829X
+    affiliation: 1  
+  - name: Nick Lesniak
+    orcid: 0000-0001-9359-5194
+    affiliation: 2
+  - name: Dana King
+    orcid: 0000-0003-4635-5272
+    affiliation: 3
+  - name: Catherine Barnier
+    orcid: 0000-0001-7074-401
+    affiliation: 1
+  - name: Matthew Flickinger
+    orcid: 0000-0002-8528-8925
+    affiliation: 4
+  - name: Jule Krüger
+    orcid: 0000-0002-2814-137X
+    affiliation: 5
+  - name: Courtney R. Armour
+    orcid: 0000-0002-5250-1224
+    affiliation: 2
+  - name: Maya M. Lapp
+    orcid: 0000-0002-8249-1186 
+    affiliation: 6
+  - name: Jason Tallant
+    orcid: 0000-0003-4790-8772
+    affiliation: 7
+  - name: Rucheng Diao
+    orcid: 0000-0002-2932-6755
+    affiliation: 1
+  - name: Morgan Oneka
+    orcid: 0000-0002-9672-0445
+    affiliation: 1
+  - name: Sarah Tomkovich
+    orcid: 0000-0003-1638-5307
+    affiliation: 2
+  - name: Jacqueline Moltzau Anderson
+    orcid: 0000-0003-1398-5980
+    affiliation: 2
+  - name: Sarah K. Lucas
+    orcid: 0000-0003-1676-5801
+    affiliation: 2
+  - name: Patrick D. Schloss^[corresponding author]
+    orcid: 0000-0002-6935-4275
+    affiliation: 2
+affiliations:
+  - name: Department of Computational Medicine & Bioinformatics, University of Michigan
+    index: 1
+  - name: Department of Microbiology & Immunology, University of Michigan
+    index: 2
+  - name: BRCF Bioinformatics Core, University of Michigan
+    index: 3
+  - name: Department of Biostatistics, University of Michigan
+    index: 4
+  - name: Center for Political Studies, Institute for Social Research, University of Michigan
+    index: 5
+  - name: Independent educator
+    index: 6
+  - name: Biological Station, University of Michigan
+    index: 7
+date: 27 Apr. 2021
+bibliography: paper.bib
+---
+
+# Summary
+
+Inspired by well-established material and pedagogy provided by The Carpentries
+[@wilson_software_2016], we developed a two-day workshop curriculum that teaches
+introductory R programming for managing, analyzing, plotting and reporting data using the R tidyverse
```

Comment:
Is it weird to say "introductory R programming....using the R tidyverse"? Maybe we should instead say:

```suggestion
introductory programming for managing, analyzing, plotting and reporting data using the R tidyverse
```

or

```suggestion
introductory R programming for managing, analyzing, plotting and reporting data using the tidyverse
```

Don't have a super strong opinion about this, but it sounds just a bit clunky to me right now.

Reply:
Pat's suggestion:

```suggestion
introductory R programming for managing, analyzing, plotting and reporting data using packages from the tidyverse
```